### ISSUE TITLE
Problem about testing result and vs pq-base methods

### ISSUE THREAD

**Issue by aruba01** [youngkyunJang/Deep-Hash-Distillation#2]
Thanks for your great works!

I noticed that the testing result of mAP in NUS-WIDE dataset is different with ITQ and SH from your papar "Self-supervised Product Quantization for Deep Unsupervised Image Retrieval - ICCV2021".

the result in this paper:
![image](https://user-images.githubusercontent.com/47864285/191806145-cf34d60f-5f42-4742-bf15-dab1d662f271.png)
the result in old paper:
![image](https://user-images.githubusercontent.com/47864285/191808102-8905d23e-5ab3-4e4a-9e6c-a5df7799fcd5.png)
I also read some other paper but they are all different. So I want to know the accurate results and reasons for this phenomenon.

Additional, I noticed that the result of this paper is much better than SPQ. Does it means unsupervised pq-based methods still has much space to improve?

**Comment by youngkyunJang** [comment]
Hi, yes unsupervised methods still have limitations especially in this benchmarks, only few training data (about 10K) are employed during training. The reason for the difference in performance is that the dataset configuration is a bit different from two benchmarks (training / test set is different).
